src/app/app-routing.module.ts

```diff
@@ -1,11 +1,17 @@
-import { NgModule } from '@angular/core';
-import { Routes, RouterModule } from '@angular/router';
+import { NgModule } from "@angular/core";
```

Comment:
use single quotes instead of double-quotes (it not a bug - just a common code style) 
It recommended using lint to check your code style

Reply:
it will fix it too.